crates/rolldown/src/module_loader/runtime_module_task.rs

```diff
@@ -1,34 +1,29 @@
+use std::sync::Arc;
+
 use arcstr::ArcStr;
 use oxc::span::SourceType;
 use oxc::{ast::VisitMut, index::IndexVec};
 use rolldown_common::{
   side_effects::DeterminedSideEffects, AstScopes, EcmaView, EcmaViewMeta, ExportsKind,
-  ModuleDefFormat, ModuleId, ModuleIdx, ModuleType, NormalModule, SymbolRef, SymbolRefDbForModule,
+  ModuleDefFormat, ModuleId, ModuleIdx, ModuleType, NormalModule, SymbolRef,
+};
+use rolldown_common::{
+  ModuleLoaderMsg, RuntimeModuleBrief, RuntimeModuleTaskResult, RUNTIME_MODULE_ID,
 };
 use rolldown_ecmascript::{EcmaAst, EcmaCompiler};
 use rolldown_error::{BuildDiagnostic, BuildResult};
 use rustc_hash::FxHashSet;
 
-use super::Msg;
 use crate::{
   ast_scanner::{AstScanner, ScanResult},
-  runtime::{RuntimeModuleBrief, RUNTIME_MODULE_ID},
   utils::tweak_ast_for_scanning::PreProcessor,
 };
 pub struct RuntimeModuleTask {
-  tx: tokio::sync::mpsc::Sender<Msg>,
+  tx: Arc<tokio::sync::mpsc::Sender<ModuleLoaderMsg>>,
```

Comment:
`Sender` is using `Arc` internally. Seems no need to wrap it with `Arc` again?

Reply:
Yeah. Here should remove `Arc`.